platform/mellanox/mlnx-sfpd/scripts/mlnx-sfpd

```diff
@@ -101,34 +100,65 @@ class MlnxSfpd:
         # open SDK API handle
         # retry at most SX_OPEN_RETRIES times to wait
         # until SDK is started during system startup
+        
         retry = 1
+        sx_api_open_pass = False
```

Comment:
>sx_api_open_pass [](start = 8, length = 16)

Should we blame the design of SDK API?
1. if api_open() is not ready, we should not start this downstream service at all
2. if api_open() is ready, all other api should be also ready.

Reply:
I can not say this is a bug of SDK or by design, this is what we can do so far.